kuma/wiki/models.py

```diff
@@ -1501,9 +1479,6 @@ def parent_trees_watched_by(self, user):
         """
         return [doc for doc in self.parents if doc.tree_is_watched_by(user)]
 
-    def get_document_type(self):
-        return WikiDocumentType
```

Comment:
Hmm, guess that's not used anymore? Might have been leftovers from elasticutils.